modules/slider/Slider.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from "react";
+import { withStyles } from "@diana-ui/base";
+import { ThemeStyleSheetFactory, WithStylesProps } from "@diana-ui/types";
+import { LabelRegular } from "@diana-ui/typography";
+
+export interface ISliderProps {
+  min: number;
+  max: number;
+  value: number;
+  step: number;
+  onChange: any;
+  disabled?: boolean;
+  className?: string;
+}
+
+export interface ISliderState {
+  leftSpacing: number;
+}
+
+const styleSheet: ThemeStyleSheetFactory = theme => ({
+  wrapper: {},
+  valueWrapper: {},
+  input: {
+    "&::-webkit-slider-thumb": {},
+    "@selectors": {
+      "&.disabled::-webkit-slider-thumb": {},
+      "&.disabled::-webkit-slider-runnable-track": {}
+    },
+    "&::-moz-range-thumb": {},
+    "&::-ms-thumb": {},
+    "&:focus": {},
+    "&::-ms-track": {},
+    "&::-webkit-slider-runnable-track": {}
+  },
+  value: {}
+});
+
+const calculateLeftSpace = (
+  totalWidth: number,
+  min: number,
+  max: number,
+  value: number
+) => (value * totalWidth) / (max - min) + 13 - (`${max}`.length * 20) / 2;
+
+const Slider: React.FC<ISliderProps & WithStylesProps> = ({
+  cx,
+  styles,
+  min,
+  max,
+  value,
+  step,
+  onChange,
+  disabled = false,
+  className = ""
```

Comment:
Is the default value necessary? Also, I'm not sure if placing the classname at the `input` level and not the `wrapper` level is the best idea, simply because I see it being used for setting margins and padding and whatnot. With the current implementation, that won't be possible.